api-schema/src/main/java/org/stellar/anchor/api/asset/Sep31Info.java

```diff
@@ -23,6 +24,8 @@ public static class ReceiveOperation {
 
     @SerializedName("max_amount")
     Long maxAmount;
+
+    List<String> methods;
```

Comment:
Do we call this `methods` here so that it's consistent with the SEP-6 asset configuration?

Reply:
Ah actually this is to match the naming is asset config, for data binding
```
    sep31:
      enabled: false
      receive:
        min_amount: 0
        max_amount: 1000000
        methods:
          - SEPA
          - SWIFT
```